cmd/qad_mbuffer_cmd.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, plugIn):
 
    def __del__(self):
       QadCommandClass.__del__(self)
-      del SSGetClass
+      del self.SSGetClass

```

Comment:
I would suggest to merge this PullRequest and create a PR for each issue (e.g. issue #40) that is fixing.
One issue => one PR